cranelift/assembler-x64/meta/src/dsl.rs

```diff
@@ -119,7 +120,7 @@ impl core::fmt::Display for Inst {
         if *has_trap {
             write!(f, " has_trap")?;
         }
-        if *custom_visit {
+        if custom.contains(CustomOperation::Visit) {
             write!(f, " custom_visit")?;
         }
```

Comment:
```suggestion
        if !custom.is_empty() {
            write!(f, " custom({custom})")?;
        }
```

Since we already have `Display` implemented for `Custom`.